src/seedu/addressbook/AddressBook.java

```diff
@@ -320,6 +328,21 @@ private static boolean isValidFilePath(String filePath) {
     private static void loadDataFromStorage() {
         initialiseAddressBookModel(loadPersonsFromFile(storageFilePath));
     }
+    
+    /**
+     * A procedure which executes the input and feedback cycle forever until a user types 'exit' and exits the program.
+     * 
+     * @param	no parameters, but user input will be scanned
+     * @return	no return value (procedure)
+     */
+    private static void infiniteInputAndFeedbackLoop() {
```

Comment:
betterName `processInputUntilExit`? The name should not give hints about the implementation.